app/components/codelijst/instance-table.js

```diff
@@ -0,0 +1,70 @@
+import Component from '@glimmer/component';
+
+import { A } from '@ember/array';
+import { action } from '@ember/object';
+import { service } from '@ember/service';
+import { tracked } from '@glimmer/tracking';
+
+export default class CodelijstInstanceTable extends Component {
+  @service store;
+
+  @tracked codelijst;
+  @tracked concepten = A();
+
+  @tracked isCodelistNameValid;
+  @tracked isConceptenChanged;
+
+  constructor() {
+    super(...arguments);
+    this.codelijst = this.args.codelijst;
+    if (this.args.concepten) {
```

Comment:
this feels scary. do we have to do things like this?

Reply:
constructor is also not enough i think, you'd have to use did-insert / did-update or better your custom modifiers: https://nullvoxpopuli.com/avoiding-lifecycle/